libdevcore/Exceptions.cpp

```diff
@@ -27,7 +27,9 @@ char const* Exception::what() const noexcept
 	if (string const* cmt = comment())
 		return cmt->c_str();
```

Comment:
`c_str()` is deprecated. Use `data()`.

Reply:
Can you address these in a new pull request?